components/script/stylesheet_loader.rs

```diff
@@ -147,6 +150,9 @@ impl FetchResponseListener for StylesheetContext {
                 }
                 StylesheetContextSource::Import(ref import) => {
                     let import = import.read();
+                    if elem.downcast::<HTMLLinkElement>().unwrap().is_alternate() {
```

Comment:
Indeed it fails. I will rebase with the change.

And indeed it say *alternate flags  Unset.* So I guess it's not a flag after all.